skbio/alignment/_util.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
@mortonjt @qiyunzhu This file is incomplete and untested, feel free to ignore for now.